docs/src/routes/components/components/forms/form-notification/test/+page.md

```diff
@@ -250,7 +250,7 @@ Het basisvoorbeeld met een invoerveld en een verzendknop.
       aria-invalid="true"
     />
     <p class="error" id="input-link-error-message">
-      <span>Foutmelding:</span> Lorem <a href="ipsum dolor">ipsum dolor</a> set amet
+      <span>Foutmelding:</span> Lorem <a href="/">ipsum dolor</a> set amet
```

Comment:
Perhaps use "#" instead of "/" to make sure people that click this link do not get lost in the 'visual example' that actually works as a link to home now.

Lacynote: this is not the only example with a workable link I think, I did not check if there are more in your PR.